templates.js

```diff
@@ -24,18 +26,27 @@ module.exports = {
         }
 
         if (exitCode in normalMap) {
-            return `&#9209;&#65039; <b>${e.Actor.Attributes.name}</b> stopped!\n${e.Actor.Attributes.image}\n${normalMap[exitCode]}`;
+            return `&#9209;&#65039; <b>${normalizeCaptainContainerName(e)}</b> stopped!\n${e.Actor.Attributes.image}\n${normalMap[exitCode]}`;
         } else if (exitCode in nonNormalMap) {
-            return `&#128308; <b>${e.Actor.Attributes.name}</b> stopped!\n${e.Actor.Attributes.image}\n${nonNormalMap[exitCode]}`;
+            return `&#128308; <b>${normalizeCaptainContainerName(e)}</b> stopped!\n${e.Actor.Attributes.image}\n${nonNormalMap[exitCode]}`;
         } else {
-            return `&#128308; <b>${e.Actor.Attributes.name}</b> stopped with exit code (${exitCode})!\n${e.Actor.Attributes.image}`;
+            return `&#128308; <b>${normalizeCaptainContainerName(e)}</b> stopped with exit code (${exitCode})!\n${e.Actor.Attributes.image}`;
         }
     },
 
     'container_health_status: healthy': e =>
-        `&#9989; <b>${e.Actor.Attributes.name}</b> healthy\n${e.Actor.Attributes.image}`,
+        `&#9989; <b>${normalizeCaptainContainerName(e)}</b> healthy\n${e.Actor.Attributes.image}`,
 
     'container_health_status: unhealthy': e =>
-        `&#9888; <b>${e.Actor.Attributes.name}</b> unhealthy!\n${e.Actor.Attributes.image}`,
+        `&#9888; <b>${normalizeCaptainContainerName(e)}</b> unhealthy!\n${e.Actor.Attributes.image}`,
 };
 
+const captainNameRegex = /^srv-captain--(.*?)\.\d+\..+/
```

Comment:
Looks like this catches some edge case. Do you think others are having the same issue?